selftests/functional/basic.py

```diff
@@ -1093,6 +1099,23 @@ def test_non_absolute_path(self):
             f"Avocado did not return rc {expected_rc}:\n{result}",
         )
 
+    @skipUnlessPathExists("/bin/bash")
+    def test_env_var_disable(self):
+        with script.TemporaryScript("exec_env_var.sh", EXEC_ENV_VARIABLE_TEST) as tst:
+            res = process.run(
+                f"env TEST_ENV=test avocado-runner-exec-test runnable-run -k exec-test -u {tst}",
+            )
+            result = res.stdout_text.split("\n")[-2]
+            self.assertIn("'returncode': 1", result, "The test might be corrupted.")
```

Comment:
It might not be obvious to someone in future what the point of this test is, so I'd be more explicit here.  Maybe `Script unexpectedly passed with environment variable set`.